napari/_qt/qt_main_window.py

```diff
@@ -287,7 +287,14 @@ def _load_window_settings(self):
         else:
             width, height = window_position
             screen_geo = QApplication.primaryScreen().geometry()
```

Comment:
Why do we select here primary screen? Why we cannot use `QApplication.screenAt(QPoint(width, height))`? 
If this functions return `None`, then we should be outside any screen.

Reply:
Ah I think that would not work because Qt does not throw an error with giving an invalid position to open napari. Just checked and indeed we can do that.